android/src/main/java/com/adyenreactnativesdk/ui/PendingPaymentDialogFragment.kt

```diff
@@ -27,13 +28,16 @@ class PendingPaymentDialogFragment() : DialogFragment() {
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
         isCancelable = false
-        return inflater.inflate(R.layout.loading, container, false)
+        return inflater.inflate(R.layout.pending_payment, container, false)
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         requireDialog().window?.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
 
+        val progress = view.findViewById<ProgressBar>(R.id.progressBar)
+        progress.visibility = if (cancelable == null) INVISIBLE else  VISIBLE
```

Comment:
you can use
```
progress.isVisible = cancellable != null
```

Reply:
Fixed here [e20b169](https://github.com/Adyen/adyen-react-native/pull/57/commits/e20b1696f849850d3759ac9b1851f95262ec63c5)